desktop/renderer/index.html

```diff
@@ -7,8 +7,17 @@
     <link rel="stylesheet" href="./fonticon.css" />
   </head>
   <body>
-    <div id="app"></div>
+    <div id="app"><div title="loading..." style="flex: 1;background-color: #f5f5f5"</div>
```

Comment:
I wanted to not make an actual UI since it causes flashes so this is a basically invisible element but allows us to understand if react loaded or not